cypress/e2e/swap/swap-page-selectors.cy.ts

```diff
@@ -0,0 +1,26 @@
+export const token = {
+  tokenIn: '#swap-currency-input .token-symbol-container',
+  tokenOut: '#swap-currency-output .token-symbol-container',
+  inputToken: '#token-search-input',
+  favoriteToken: '[data-testid="favorite-token"]',
```

Comment:
```suggestion
  favoriteToken: '[data-testid=favorite-token]',
```
You done need those `"`

Reply:
Thanks. I've updated.